scripts/apps/authoring/views/authoring-topbar.html

```diff
@@ -249,8 +249,12 @@
                         {{ :: 'Desks' | translate }}
                       </button>
                       <ul class="dropdown__menu dropdown__menu--submenu-left" sd-mark-desks-dropdown></ul>
+
                     </div>
                   </li>
+                  <li ng-repeat="action in extraActionsFromExtensions">
```

Comment:
It doesn't work when there is no group specified or another group is specified. You can leave it unimplemented, but you have to state it in the comments of the interface. It shouldn't be too hard to at least an ability to add ungrouped items.

Reply:
How about showing them on top (no groups) https://github.com/superdesk/superdesk-client-core/pull/3051/files#diff-b8516f0f8a0f9b0d4d13ad9354c960d3R235-R241

Reply:
I'd put it at the [bottom of currently ungrouped items](https://user-images.githubusercontent.com/2076953/62716832-19adce80-ba03-11e9-918b-31485d456314.png)

Reply:
@tomaskikutis yeah that's where it is, I didn't explain it properly